ak_vendor/locale/ja/LC_MESSAGES/django.po

```diff
@@ -11268,15 +11303,18 @@ msgstr ""
 
 #: vulnerabilities.py:5262
 msgid "Stack Trace Enabled"
-msgstr ""
+msgstr "スタックトレースが有効"
 
 #: vulnerabilities.py:5263
 msgid "Does the application leaks out sensitive information via Stacktrace?"
-msgstr "このアプリケーションはCoreDataに機密情報を保存していませんか？"
+msgstr ""
+"このアプリケーションはスタックトレースを通じて機密情報を漏えいしていません"
+"か？"
 
 #: vulnerabilities.py:5264
 msgid "The application does not leak senstive information via stacktrace"
-msgstr "このアプリケーションはCoreDataに機密情報を保存していません。"
+msgstr ""
+"このアプリケーションはスタックトレースを通じて機密情報を漏えいしていません。"
```

Comment:
@Praseetha-KR this is wrong. no new line should be there

Reply:
@subho007 `makemessages` is auto-formatting .po file which added the new line ^